build.gradle

```diff
@@ -122,8 +122,16 @@ dependencies {
     // need to use snapshots as the stable version is from 2013 and doesn't support v1.0.1 CitationStyles
     compile 'org.citationstyles:styles:1.0.1-SNAPSHOT'
     compile 'org.citationstyles:locales:1.0.1-SNAPSHOT'
+    compile 'de.undercouch:citeproc-java:0.6'
```

Comment:
I didnt change sth in this gradle file, i just deleted my dependency i added last time. it should be the original from your master branch. i will investigate why there is a change, this is not with purpose.